web-portal/pages/donate.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="container info-page">
+    <h1>Donate</h1>
+
+    <p>
+        <em>Why we need the funds?</em> We are sustained by a mix of volunteer and paid activities. We try to pay our interns for doing research and outreach tasks and generally behave like solid human beings. We are also very much interested in ramping up our outreach activities to grow our positive impact on folks around us - so in the not-too-distant future, this means hiring a dedicated staff person. Also, occasionally we commission artists to create stickers and other materials we use for promotion.
+    </p>
+    <p>
+        <a href="https://www.paypal.com/biz/fund?id=GCF56FD8ARSD6" target="_new">Please donate via PayPal</a>
+    </p>
+
+  </div>
+</template>
+
+<script>
+  export default {
+    data: function () {
+      return {
+        show_blank: true
+      }
+    },
+    methods: {
+      FillInTheBlank: function (status) {
+        if (status === 'ON') {
+          this.show_blank = false
+        } else {
+          this.show_blank = true
+        }
+      }
+    }
```

Comment:
This data function and method aren't needed and can be removed.

In fact, the entire `script` block could be removed, but we include the empty block in some of the other static pages, so I guess it makes sense to keep it for consistency.